src/js/core/methods/helpers/cardanoTokens.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+import { validateParams } from '../helpers/paramsValidator';
+
+import type { CardanoTokenGroupType, CardanoTokenAmountType } from '../../../types/trezor/protobuf';
+import type { CardanoTokenGroup, CardanoTokenAmount } from '../../../types/networks/cardano';
+
+export const validateTokenBundle = (tokenBundle: CardanoTokenGroup[]) => {
+    console.log(tokenBundle)
```

Comment:
Not needed.

Reply:
:facepalm: forgot debugging statement, thanks for pointing out